corehq/apps/geospatial/static/geospatial/js/gps_capture.js

```diff
@@ -46,9 +46,18 @@ hqDefine("geospatial/js/gps_capture",[
             let latNum = parseFloat(self.lat());
             let lonNum = parseFloat(self.lon());
 
-            const latValid = (!isNaN(latNum) && latNum >= -90 && latNum <= 90) || !self.lat().length;
+            // If the text is a number followed by text (e.g. 15foobar) then parseFloat() will not show as NaN
```

Comment:
I think good to update the comment to say something like "then parseFloat() will ignore the training text (e.g. 15) so we check for length to catch such cases"

Reply:
looks like something went wrong when I was typing this, what I intended to say was
"...training text (e.g. 15foobar becomes 15) so ..."

Reply:
Ah, well I've updated the example in the comment to make it a bit clearer (e.g. 15foobar becomes 15). The rest of the comment I've left as is, since I don't think it needs to be as long as I orignally had it.

Reply:
Thanks @zandre-eng

P.S: I would recommend making use of `--amend` when it seems a new commit is not necessary from the PR perspective and when feasible (when commit not already on staging / sharing branches with others) to reduce number of commits.

Reply:
You could also consider merging commits later as well. The only challenge with that it needs that no "master update" commit is in between them.

Reply:
I think Github retains comments, they just don't show on the files, not entirely sure.

Reply:
`--amend` is great during development, just noting that amending/rebasing (anything that requires a force push) isn't advisable once review has started, because it basically forces re-review of the entire PR.

Reply:
> because it basically forces re-review of the entire PR

Oh, I would think it would be just the commits at and after the amend.

Either way, agreed that it should be used with caution and always highlighted to reviewer if done.